src/test/ui/impl-trait/no-method-suggested-traits.stderr

```diff
@@ -205,30 +268,55 @@ error[E0599]: no method named `method3` found for type `std::rc::Rc<&mut std::bo
     |
 126 |     std::rc::Rc::new(&mut Box::new(&1_usize)).method3(); //~ ERROR no method named
     |                                               ^^^^^^^
+    | 
+   ::: /checkout/src/liballoc/rc.rs
+    |
+284 | pub struct Rc<T: ?Sized> {
+    | ------------------------ method `method3` not found for this
```

Comment:
We should avoid pointing at ADTs in `std`.

Reply:
Pointing to any other crate isn't very helpful imo. `std` isn't special. Just giving the full module path to them should suffice.